packages/mc-html-template/src/process-headers.js

```diff
@@ -24,13 +24,20 @@ const mergeCspDirectives = (...csps) =>
       ),
     {}
   );
-const toHeaderString = (directives = {}, seperator = '; ') =>
+const toHeaderString = (directives = {}) =>
   Object.entries(directives)
     .map(
       ([directive, value]) =>
         `${directive} ${Array.isArray(value) ? value.join(' ') : value}`
     )
-    .join(seperator);
+    .join('; ');
+const toStructuredHeaderString = (directives = {}) =>
```

Comment:
I wouldn't mind to have a specific function for mapping the permissions policy directives. Then it's clear what it does, especially the part with the `=`.

Reply:
And by that I mean naming the function specific for the permissions policy header.

Reply:
Yes and no. The format isn't specific to it. It's a structured header to how I understand the spec. I think having a mapper per header type not value makes sense. At least to me. Now I know that a permissions policy header is a structured header by reading the code.

Reply:
I didn't read the spec. What does "structured header" mean exactly?

Reply:
To what I understand it's the format: `key=(), key2=(), key3=()`.

Reply:
Ok got it, I did some googling and that's the name of this data structure. Thanks

https://www.w3.org/TR/permissions-policy-1/#permissions-policy-http-header-field
<img width="882" alt="image" src="https://user-images.githubusercontent.com/1110551/116864183-bec0ff80-ac07-11eb-9ca5-1135561ff671.png">